src/kryptos/k4/geometry_combined_sweep.py

```
@@ -0,0 +1,234 @@
+            "total_tested": total_tested,
+            "positional_eureka_threshold": positional_eureka_threshold,
+            "keyword_eureka_threshold": keyword_eureka_threshold,
+            "ts_start": ts_start,
+        },
+        "best_candidates": best_candidates[:10],
+        "null_artifact_path": str(Path(null_artifact_path).resolve()),
+    }
+    Path(null_artifact_path).write_text(json.dumps(summary, indent=2, default=str), encoding="utf-8")
+
+    graph = hypothesis_graph.load(graph_path)
+    hypothesis_graph.record_result(graph, _TRANSFORM_EDGE, "null", evidence=str(Path(null_artifact_path).resolve()))
```

Pylint (4.0.7)

[convention] Line too long (116/88)

(C0301)